recipes/nginx.rb

```diff
@@ -76,3 +87,48 @@
 nginx_site 'nagios3.conf' do
   notifies :reload, 'service[nginx]'
 end
+
+node.default['nagios']['web_user'] = node['nginx']['user']
+node.default['nagios']['web_group'] = node['nginx']['group']
+
+# configure the appropriate authentication method for the web server
+case node['nagios']['server_auth_method']
+when 'openid'
```

Comment:
Same question as with apache2: do we really want to support openid at all anymore?